CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Make ./tests/syntax-tests/regression_test.sh work on recent versions of macOS, see #1443 (@Enselic)
 - VimL syntax highlighting fix, see #1450 (@esensar)
 - Print an 'Invalid syntax theme settings' error message if a custom theme is broken, see #614 (@Enselic)
+- Ignore PAGER=most with a warning to stderr, but allow override with BAT_PAGER or --config, see #1063 (@Enselic)
```

Comment:
maybe: "… or `--pager` from the config file, see …"

Reply:
Sorry about the typo, should have been `--pager`, not `--config`. I completely reworded the entry now though, please take a look.